src/components/ItaliaTheme/Blocks/IconBlocks/Block/ViewBlock.jsx

```diff
@@ -65,6 +65,11 @@ const ViewBlock = ({ data, isOpen, toggle, id, index }) => {
             tag={UniversalLink}
             href={data.href ?? '#'}
             text={data.linkMoreTitle || intl.formatMessage(messages.vedi)}
+            aria-label={
+              data.linkMoreTitle
+                ? data.linkMoreTitle
+                : intl.formatMessage(messages.vedi) + ' ' + data.title
```

Comment:
```suggestion
                : `${intl.formatMessage(messages.vedi)}${data.title ? ` ${data.title}` : ''}`
```
Vedendo sopra data.title non è scontato ci sia sempre

Reply:
Vero, inoltre renderizzare il title si fa con redraft come sopra alla riga 46

```jsx
            {redraft(
              data.title,
              config.settings.richtextViewSettings.ToHTMLRenderers,
              config.settings.richtextViewSettings.ToHTMLOptions,
            )}
```

Farei inoltre come suggerisce giulia nella descrizione del ticket, cioè il data.title ce lo metterei sempre
```jsx
aria-label={`${data.linkMoreTitle || intl.formatMessage(messages.vedi)} ${data.title ? redraft(...) : ''}`}
```